google/cloud/security/common/data_access/organization_dao.py

```diff
@@ -59,8 +58,8 @@ def get_org_iam_policies(self, timestamp):
                     org = Organization(organization_id=row[0])
                     iam_policy = json.loads(row[1])
                     org_iam_policies[org] = iam_policy
-                except ValueError as json_error:
-                    self.logging.warn('Error parsing json {}'.format(row[2]))
+                except ValueError as _:
```

Comment:
You can just say 'except ValueError' without assigning a variable if you don't use it.